bicep/api.bicep

```diff
@@ -0,0 +1,28 @@
+param appName string
+param appServicePlanId string
+param env string
+param location string
+param team string
+
+var appServiceName = '${env}-${team}-${appName}-api-${location}'
+
+resource appService 'Microsoft.Web/sites@2021-02-01' = {
```

Comment:
I'd love to learn more about these

Reply:
Will definitely have to do an ADR once these are finished